packages/interface/src/components/pinned-items/index.js

```diff
@@ -1,34 +1,24 @@
 /**
  * External dependencies
  */
-import { isEmpty } from 'lodash';
 import classnames from 'classnames';
 
 /**
- * WordPress dependencies
+ * Internal dependencies
  */
-import { Slot, Fill } from '@wordpress/components';
+import ActionItem from '../action-item';
 
 function PinnedItems( { scope, ...props } ) {
-	return <Fill name={ `PinnedItems/${ scope }` } { ...props } />;
+	return <ActionItem name={ `PinnedItems/${ scope }` } { ...props } />;
 }
 
 function PinnedItemsSlot( { scope, className, ...props } ) {
 	return (
-		<Slot name={ `PinnedItems/${ scope }` } { ...props }>
-			{ ( fills ) =>
-				! isEmpty( fills ) && (
-					<div
-						className={ classnames(
-							className,
-							'interface-pinned-items'
-						) }
-					>
-						{ fills }
-					</div>
-				)
-			}
-		</Slot>
+		<ActionItem.Slot
```

Comment:
Doesn't it make code harder to follow? The existing implementation is very simple so maybe we can leave it as is. 

If we were to leave it as is, shouldn't we propagate `scope` to the slot?

Reply:
>Doesn't it make code harder to follow? The existing implementation is very simple so maybe we can leave it as is.

I reverted to the previous implementation 👍